src/buckets/components/BucketCardMeta.tsx

```diff
@@ -57,10 +57,22 @@ const BucketCardMeta: FC<Props> = ({bucket, notify}) => {
       {schemaLabel}
     </span>
   )
+  const bucketID = (
+    <CopyToClipboard
+      text={bucket.id}
+      onCopy={handleCopyAttempt}
+      key={bucket.id}
+    >
+      <span className="copy-bucket-id" title="Click to Copy to Clipboard">
+        ID: {bucket.id}
+        <span className="copy-bucket-id--helper">Copy to Clipboard</span>
+      </span>
+    </CopyToClipboard>
+  )
 
   const bucketInfo = CLOUD
-    ? [persistentBucketMeta, schemaBlock]
-    : [persistentBucketMeta]
+    ? [persistentBucketMeta, schemaBlock, bucketID]
+    : [persistentBucketMeta, bucketID]
```

Comment:
does this array structure generate a key warning in the console?

Reply:
it did! i had to add a key inside `CopyToClipboard` to get rid of the warning